### PR TITLE
vict-5

### LINKED ISSUE #7: vict-5: [Basic Create] - Implement sending data after case creation in front (8h)
Modify create case page functionality:
- Create POST method in controller:
    - now it just prints case in console
- Data from form is sent to POST: **/cases/new** by clicking button "Створити".
- Validation is added:
    - "**JUDGE**", "**NUMBER**", "**OPPONENT**", "**SUBJECT**" fields cannot be empty
    - validation is implemented like in theme: [https://bootstrapmade.com/demo/NiceAdmin/](https://bootstrapmade.com/demo/NiceAdmin/)
    - After creation of case redirect to listing

Acceptance criteria:
- dummy POST method is created
- data is passed to POST method of controller
- validation is implemented
- after creation of case there is a redirect to listing

### PR DESCRIPTION
**Pull request by Ratchet34702**

### CODE REVIEW

File: src/main/resources/views/newCase.rocker.html [hunk 1]
```diff
@@ -6,51 +6,51 @@
...
-            <form>
+            <form action="/cases/new" method="post">
                 <div class="form-group">
                     <label for="currentStage">Стадія розгляду</label>
-                    <input type="text" class="form-control" id="currentStage">
+                    <input type="text" class="form-control" id="currentStage" name="currentStage">
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Дата і час</label>
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: In label element, write correspondent column in "for". When specifying id of input, end with "Id" word. For example: "currentStageId".

File: src/main/resources/views/newCase.rocker.html [hunk 2]
```diff
@@ -6,51 +6,51 @@
...
-                    <input type="text" class="form-control" id="stageDeadline">
+                    <input type="text" class="form-control" id="currentStageDeadline" name="currentStageDeadline">
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Суд / суддя</label>
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 3]
```diff
@@ -6,51 +6,51 @@
...
-                    <input type="text" class="form-control" id="judge">
+                    <input type="text" class="form-control" id="judge" name="judge" required="required">
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 4]
```diff
@@ -6,51 +6,51 @@
...
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Номер Справи</label>
-                    <input type="text" class="form-control" id="number">
+                    <input type="text" class="form-control" id="number" name="number" required="required">
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 5]
```diff
@@ -6,51 +6,51 @@
...
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Опонент</label>
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 6]
```diff
@@ -6,51 +6,51 @@
...
-                    <input type="text" class="form-control" id="opponent">
+                    <input type="text" class="form-control" id="opponent" name="opponent" required="required">
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Предмет спору</label>
-                    <input type="text" class="form-control" id="subject">
+                    <input type="text" class="form-control" id="subject" name="subject" required="required">
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 7]
```diff
@@ -6,51 +6,51 @@
...
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Клієнт</label>
-                    <input type="text" class="form-control" id="client">
+                    <input type="text" class="form-control" id="client" name="client">
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 8]
```diff
@@ -6,51 +6,51 @@
...
                 </div>
 
                 <div class="form-group">
                     <label for="currentStage">Примітка</label>
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 9]
```diff
@@ -6,51 +6,51 @@
...
-                    <input type="text" class="form-control" id="note">
+                    <input type="text" class="form-control" id="note" name="note">
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

File: src/main/resources/views/newCase.rocker.html [hunk 10]
```diff
@@ -6,51 +6,51 @@
...
                 </div>
 
 
                 <div class="form-group">
                     <label for="doneWork">Проведена робота</label>
-                    <textarea class="form-control" id="doneWork" rows="3"></textarea>
+                    <textarea class="form-control" id="doneWork" name="doneWork" rows="3"></textarea>
```
Comment (makskostyshen) [inline src/main/resources/views/newCase.rocker.html]: and here

**Review by makskostyshen — COMMENTED** [review]
Good job. But minor style fixes are needed

**Comment by Ratchet34702** [comment]
Worked 40 minutes:

- Resolve problems with project on PC
- Fix style issues

**Comment by makskostyshen** [comment]
great!

**Review by makskostyshen — APPROVED** [review]

**Pull request merged**